plugins/k8sauditeks/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/falcosecurity/plugins/plugins/k8saudit-eks
```

Comment:
This is not consitent with the name of the directory. Consider my naming suggestion above here too.